cherrypy/lib/static.py

```diff
@@ -28,6 +29,21 @@ def _setup_mimetypes():
 _setup_mimetypes()
 
 
+def _utf8_content_disposition(disposition, file_name):
+    """Create HTTP header for downloading a file with UTF-8 filename.
+    See this and related answers:
+    https://stackoverflow.com/a/8996249/2173868
+    """
+    ascii_name = (
+        unicodedata.normalize('NFKD', file_name).
+        encode('ascii', errors='ignore').decode()
+    )
+    quoted_name = urllib.parse.quote(file_name)
+    header = u'{}; filename="{}"'.format(disposition, ascii_name)
+    header += u'; filename*=UTF-8\'\'{}'.format(quoted_name)
```

Comment:
>    o  Include a "filename*" parameter where the desired filename cannot
      be expressed faithfully using the "filename" form.  Note that
      legacy user agents will not process this, and will fall back to
      using the "filename" parameter's content.

Looks like `filename*` should only be included when different from `filename`.

Reply:
Yeah, this is true, i forgot it, that`s why in first pull request there was a condition before second part of header variable assignment, I will return it